graph/global-defaults.yaml

```diff
@@ -0,0 +1,39 @@
+# Convenience definitions, use as eg. $commit
```

Comment:
We need sperate files for windows and linux

Reply:
How do we determine what kind of environment the task will be operating in?/Are most  of these images not multi-arch?